src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,45 @@
+.stars {
+  display: flex;
+
+  &__star {
+    background: url(../images/star.svg) center no-repeat;
+    width: 16px;
+    height: 16px;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--0 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--1 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--2 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--3 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--4 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--5 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--6 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
+
+  &--7 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```

Comment:
Fix this part according previous stars task 
![image](https://github.com/mate-academy/layout_catalog/assets/70009205/1af8aff4-a840-4c40-b2a1-a00c120059db)
```suggestion
  &--1 &__star:nth-child(-n+1),
  &--2 &__star:nth-child(-n+2),
  &--3 &__star:nth-child(-n+3),
  &--4 &__star:nth-child(-n+4),
  &--5 &__star:nth-child(-n+5) {
    background-image: url(../images/star-active.svg);
  }
```